models.py

```diff
@@ -25,10 +25,10 @@ class Meta:
         ordering = ['name']
 
     def books(self):
-        return Book.search(query='name:"%s"' % self.name )
+        return Book.search(query='genre_aat:books+AND+name:"%s"' % self.name )
 
     def prints(self):
-        return Print.search(query='contributor:"%s"' % self.name )
+        return Book.search(query='(genre_aat:"etchings (prints)"+OR+genre_aat:"engravings (prints)")+AND+name:"%s"' % self.name )
```

Comment:
This doesn't feel right - to have the prints() method return a bunch of Book objects. Would it work to return Prints here? Do we need to update the Print class?